doc/guides/coverage.rst

```diff
@@ -127,27 +127,23 @@ You may postprocess these with your preferred tools. For example:
 Sanitycheck coverage reports
 ============================
 
-When targeting boards based on the POSIX architecture,
 Zephyr's :ref:`sanitycheck script <sanitycheck_script>` can automatically
 generate a coverage report from the tests which were executed.
-You just need to invoke it with the ``-C`` command line option.
+You just need to invoke it with the ``--coverage`` command line option.
 
-For example you may run::
+For supported emulator targets it's enough to just specify ``--coverage``::
 
-    $ sanitycheck -p native_posix -T tests/kernel -C
+    $ sanitycheck --coverage -p qemu_x86 -T tests/kernel
 
 which will produce ``sanity-out/coverage/index.html`` with the report.
 
-For emulated targets you will need to pass the correct ``gcov`` binary from
-the SDK, the default of using the ``gcov`` found in the default path is likely
-not going to work. For example, to produce reports using the
-``qemu_x86_coverage`` target::
-
-    $ sanitycheck --coverage --coverage-platform=qemu_x86_coverage -p qemu_x86_coverage --gcov-tool $ZEPHYR_SDK_INSTALL_DIR/i586-zephyr-elf/bin/i586-zephyr-elf-gcov
+To produce coverage reports based on the ``native_posix`` target, you will need
```

Comment:
Could you maybe phrase it as something like:
"To produce coverage reports based on ``native_posix`` or other POSIX arch based targets, you will" 

Background: The `nrf52_bsim` is also based on the posix arch, and it is quite useful to get coverage reports from it.

Reply:
done